docs/source/zero-to-binderhub/setup-binderhub.rst

```diff
@@ -325,14 +388,32 @@ JupyterHub. Now, add the following lines to ``config.yaml`` file::
     BinderHub:
       hub_url: http://<IP in EXTERNAL-IP>
 
-Next, upgrade the helm chart to deploy this change::
+When using ``Ingress``
+~~~~~~~~~~~~~~~~~~~~~~
+
+If JupyterHub is exposed using ``Ingress``,
+you can use any of the domains that JupyterHub is answering.
```

Comment:
What do you mean by this? Only `jupyterhub.XXX.XXX.XXX.XXX.nip.io` is configured.